DNN Platform/Connectors/GoogleAnalytics/GoogleAnalyticsConnector.cs

```diff
@@ -165,28 +173,27 @@ public bool SaveConfig(int portalId, IDictionary<string, string> values, ref boo
 
                 string trackingID;
                 string urlParameter;
-                bool trackForAdmin;
-                bool anonymizeIp;
-                bool trackUserId;
+                string trackForAdmin;
+                string anonymizeIp;
+                string trackUserId;
 
                 isValid = true;
 
                 if (isDeactivating)
                 {
                     trackingID = null;
                     urlParameter = null;
-                    trackForAdmin = false;
-                    anonymizeIp = false;
-                    trackUserId = false;
+                    trackForAdmin = null;
+                    anonymizeIp = null;
+                    trackUserId = null;
                 }
                 else
                 {
-                    trackingID = values["TrackingID"] != null ? values["TrackingID"].ToString().Trim() : String.Empty;
-                    urlParameter = values["UrlParameter"] != null ? values["UrlParameter"].ToString().Trim() : String.Empty;
-
-                    bool.TryParse(values["TrackAdministrators"].ToLowerInvariant(), out trackForAdmin);
-                    bool.TryParse(values["AnonymizeIp"].ToLowerInvariant(), out anonymizeIp);
-                    bool.TryParse(values["TrackUserId"].ToLowerInvariant(), out trackUserId);
+                    trackingID = values["TrackingID"] != null ? values["TrackingID"].ToLowerInvariant().Trim() : string.Empty;
+                    urlParameter = values["UrlParameter"] != null ? values["UrlParameter"].ToLowerInvariant().Trim() : string.Empty;
```

Comment:
```suggestion
                    urlParameter = values["UrlParameter"]?.Trim() ?? string.Empty;
```

Reply:
Any reason for removing the .ToLowerInvariant or is that a mistake?

Reply:
It seemed that `ToLowerInvariant` was added to make sure that `True` was turned into `true` for the Boolean settings, but I don't see any reason to make the values of the two text settings always be lower case.  I assumed it was just copied from the Boolean case without realizing.